src/lib/ApiConnection.js

```diff
@@ -16,6 +16,9 @@ import ErrorPage from 'pages/_error.js';
 
 const { serverRuntimeConfig, publicRuntimeConfig } = getConfig();
 
+const ds = publicRuntimeConfig.DISABLE_SUBSCRIPTIONS;
+const disableSubscriptions = ((!(ds == null) && ds.toLowerCase()) || false) === 'true';
```

Comment:
Can probably just simplify this to `const disableSubscriptions = ds?.toLowerCase() === 'true';`

Reply:
thats a bit fancy